lib/langchain/vectorsearch/qdrant.rb

```diff
@@ -44,14 +44,14 @@ def find(ids: [])
     # Add a list of texts to the index
     # @param texts [Array<String>] The list of texts to add
     # @return [Hash] The response from the server
-    def add_texts(texts:, ids: [])
+    def add_texts(texts:, ids: [], payload: {})
       batch = {ids: [], vectors: [], payloads: []}
 
       Array(texts).each_with_index do |text, i|
         id = ids[i] || SecureRandom.uuid
-        batch[:ids].push(id)
+        batch[:ids].push(id)        
         batch[:vectors].push(llm.embed(text: text).embedding)
-        batch[:payloads].push({content: text})
+        batch[:payloads].push(payload.present? ? {content: text}.merge!(payload) : {content: text})
```

Comment:
I think this can be simplified with:
```ruby
{ content: text }.merge(payload)
```
Because even if `payload` is `{}`, it'll just work.

Reply:
Yes, you're right on that. I will fix that.